tests/lcs/core/strings/byteoffset.livecodescript

```diff
@@ -0,0 +1,36 @@
+script "CoreStringsByteoffset"
+/*
```

Comment:
I suggest adding explicit tests for empty needle in empty haystack, and empty needle in non-empty haystack too.